modules/revcontentBidAdapter.js

```diff
@@ -0,0 +1,290 @@
+// jshint esversion: 6, es3: false, node: true
+'use strict';
+
+import {registerBidder} from '../src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+import {ajax} from '../src/ajax';
+
+const BIDDER_CODE = 'revcontent';
+const NATIVE_PARAMS = {
+  title: {
+    id: 0,
+    name: 'title'
+  },
+  image: {
+    id: 3,
+    type: 3,
+    name: 'img'
+  },
+  sponsoredBy: {
+    id: 5,
+    name: 'data',
+    type: 1
+  }
+};
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: ['banner', 'native', 'video'],
+  isBidRequestValid: function (bid) {
+    return (typeof bid.params.apiKey !== 'undefined' && typeof bid.params.userId !== 'undefined' && bid.hasOwnProperty('nativeParams'));
```

Comment:
so no matter what each bid needs `nativeParams` ?

Is revcontent only doing native ads?

Or am I missing something?

Reply:
Yep only native at this time. I am going to update `supportedMediaTypes` to reflect.